tgui/packages/tgui/interfaces/_FaxMachine.js

```diff
@@ -0,0 +1,286 @@
+                      Message from {received_paper.source}:
+                    </span>
+                    <BlockQuote mt={1}>
+                      {received_paper.contents}
+                    </BlockQuote>
+                  </Box>
+                ) : (
+                  <Box>
+                    <i>
+                      No papers have been received.
+                    </i>
+                  </Box>
+                ))}
+            </Stack.Item>
+            <Stack.Item>
+              {tab === 1 && stored_paper && (
+                <Stack>
+                  <Stack.Item>
+                    <Button
+                      height="100%"
+                      icon="fax"
+                      color={emagged
+                        ? "bad" : "good"}
+                      content="Send to: "
+                      disabled={tab !== 1
+                        || !stored_paper
+                        || !(can_send_cc_messages || emagged)}
+                      tooltip={"Send the contents of the paper currently inserted \
+                      in the machine to the destination specified. Response not guaranteed. \
+                      A copy of the sent paper will print, too - for record-keeping."}
+                      onClick={() => act('send_stored_paper', {
+                        destination_machine: destination,
+                      })} />
+                  </Stack.Item>
+                  <Stack.Item grow>
+                    <Dropdown
+                      width="100%"
+                      height="100%"
+                      selected={selectedDestination}
+                      options={destination_options}
+                      onSelected={dest => { setDestination(dest); }} />
+                  </Stack.Item>
+                </Stack>
+              )}
+              {tab === 2 && received_paper && (
+                <Stack>
+                  <Stack.Item>
+                    <Button
+                      disabled={!received_paper}
+                      content="Print received Fax"
+                      tooltip="Print the last received fax."
+                      onClick={() => act('print_received_paper')} />
+                  </Stack.Item>
+                </Stack>
+              )}
+            </Stack.Item>
+          </Stack>
+        </Section>
+        <Section
+          title="Paperwork"
+          buttons={(
+            <Button.Checkbox
+              checked={can_receive}
+              content="Toggle Incoming Paperwork"
+              tooltip={(can_receive ? "Disable" : "Enable")
+                + " the ability for this fax machine \
+                to receive paperwork every five minutes."}
+              onClick={() => act('toggle_recieving')}
+            />
+          )}>
+          <Stack vertical grow>
+            <Stack.Item height={2}>
+              {received_paperwork && received_paperwork.length > 0 ? (
+                <Tabs>
```

Comment:
`received_paperwork` will always exist, so only the length needs to be checked.
(Either `ui_data()` supplies it, or it defaults to `[]` on L11)
```suggestion
            <Stack.Item height={2}>
              {received_paperwork.length > 0 ? (
                <Tabs>
```